omod/src/main/webapp/mapLocations.jsp

```diff
@@ -32,6 +34,14 @@
 			<tr>
 				<td />
 				<td><input value="Map Locations" name="submit" type="submit"></td>
+                                </c:if>
+                                    <c:if test="${empty orgunits}">
+                                    <div id="openmrs_msg"><spring:message code="dhisreport.orgUnitCodeDoesNotExist" /> </div>
+                                </td>
+                                 <tr>
+ 				<td />
+                                <td><input value="Map Locations" name="submit" type="submit" disabled></td>
```

Comment:
@dkayiwa yes the disabled button is supposed to be under the if condition and I fixed the URL. Sorry about that.